sdk/eventhub/event-hubs/src/pumpManager.ts

```diff
@@ -41,6 +41,13 @@ export interface PumpManager {
    */
   isReceivingFromPartition(partitionId: string): boolean;
 
+  /**
+   * Returns a list of partition ids that the pump manager is actively receiving events from.
+   * @ignore
+   * @internal
+   */
+  receivingFromPartitions(): string[];
```

Comment:
Where is this used?

Reply:
Ah sorry, I was using this earlier on but then stopped using it. I'll remove.

Reply:
Removed!